packages/web/src/graphql/queries/User.js

```diff
@@ -0,0 +1,28 @@
+import gql from "graphql-tag";
+
+export const GET_USER_QUERY = gql`
+    query getUser($_id: ID!) {
+        user(_id: $_id) {
+            _id
+            author {
+                _id
+                email
+            }
+        }
+    }
+`;
+
+export const GET_ALL_USERS_QUERY = gql`
+    query getAllUsers {
+        users {
+            _id
+            email
+            posts {
+                _id
+                title
+                body
+                published
+            }
+        }
+    }
+`;
```

Comment:
Using the mutations in different files just make the dev experience worse and slower in my opinion, any thoughts?

Reply:
Let's imagine if we had 25 different mutations, it would be worse to import it every time we need one single mutation. Put all the mutations in one single file helps to get a better project structure, in a huge project we could also split the mutations by types like User mutations, Post mutations, etc.

Reply:
i can't imagine a scenario that we have 25 different mutations in 1 screen. Edge cases don't rule our dev experience.

Reply:
I think that we should separate the concerns, in some cases we might need to use a mutation over and over again, that's why separate our mutations and queries by file would be good.

Reply:
can u describe an example?

Reply:
For example the _users_ query, in some case we could and want to use it in another screen, but if we do not have our queries in one single file, we would waste a lot of time searching for it, and also it wouldn't be the best approach. 

Any developer that starts to use the project would know that our queries and mutations are in one single file, with that they could understand better and more quickly how it works. If we maintain the queries and mutations in specific files would be harder to find.